frontend/micro-ui/web/micro-ui-internals/packages/modules/sv/src/pages/citizen/Create/index.js

```diff
@@ -94,12 +93,10 @@ const SVCreate = ({ parentRoute }) => {
 
   const onSuccess = () => {
     clearParams();
-    queryClient.invalidateQueries("SV_CREATE");
+    queryClient.invalidateQueries("SV_CREATES");
+    sessionStorage.removeItem("beneficiary");
```

Comment:
Please add comments to every method as per the standard then I will merge

Reply:
Comments added